server/service/handler.go

```diff
@@ -233,6 +234,9 @@ func MakeHandler(svc fleet.Service, config config.FleetConfig, logger kitlog.Log
 	fleetHandlers := makeKitHandlers(fleetEndpoints, fleetAPIOptions)
 
 	r := mux.NewRouter()
+	if config.Logging.TracingEnabled && config.Logging.TracingType == "opentelemetry" {
```

Comment:
The size limit is 16KB of base64 encoded data. Its fine for now, but once we start needing more advanced configuration with several services like SSM, managing individual users instead of just ec2-user, ect you quickly run out of bytes in user-data.

Reply:
It also means that we can standardize deployments for all EC2 stuff we might use in the future (for EKS if we want that, or ECS with spot instances, running mysql outside of RDS for loadtesting, etc).